extensions/wikia/WikiFactory/api/ApiWikiFactorySaveVariable.php

```diff
@@ -6,7 +6,7 @@ class ApiWikiFactorySaveVariable extends ApiBase {
 	 * @throws UsageException
 	 */
 	public function execute() {
-		if ( !$this->getUser()->isAllowed( 'wikifactory' ) ) {
+		if ( !$this->isInternalRequest() && !$this->getUser()->isAllowed( 'wikifactory' ) ) {
```

Comment:
Would this be more clear as `!($this->isInternalRequest() || $this->getUser()->isAllowed( 'wikifactory' ))`?

Reply:
I think that's a matter of personal preference. Personally, I think the less parens the easier it is to read code, especially here where I'd have to mentally expand the `!` to the two conditions.